src/mudlet.pro

```diff
@@ -384,16 +384,8 @@ exists("$${_PRO_FILE_PWD_}/../3rdparty/edbee-lib/edbee-lib/edbee-lib.pri") {
     error("Cannot locate edbee-lib editor widget submodule source code, build abandoned!")
 }
 
-win32 {
-    APPVEYOR_TEST = $$(APPVEYOR)
-    !isEmpty( APPVEYOR_TEST ) {
-        equals(APPVEYOR_TEST, "True") {
-            # We are running in the AppVeyor CI environment
-            !exists("$${_PRO_FILE_PWD_}/../3rdparty/edbee-lib/edbee-lib/edbee-lib.pri") {
-                error("Cannot locate lua-code-formatter submodule source code, build abandoned!")
-            }
-        }
-    }
+!exists("$${_PRO_FILE_PWD_}/../3rdparty/lcf/lcf-scm-1.rockspec") {
+    error("Cannot locate lua code formatter submodule source code, build abandoned!")
```

Comment:
How come? The formatter is not essential.

Reply:
It is - the context menu item provided in the C++ core has to have this to work.